### PR TITLE
[plugin.video.zoneminder] 2.0.0

### PR DESCRIPTION
**Pull request by petegallagher**
### Description
New plugin to display live feeds from [ZoneMinder|https://zoneminder.com/] video surveillance software in Kodi.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.zoneminder/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.zoneminder" name="ZoneMinder" version="2.0.0" provider-name="Peter Gallagher">
+  <requires>
+    <import addon="xbmc.python" version="2.19.0"/>
+    <import addon="script.module.requests" version="2.3.0"/>
+    <import addon="script.module.simplejson" version="2.0.0"/>
```
Comment (enen92) [inline plugin.video.zoneminder/addon.xml]: not needed anymote

File: plugin.video.zoneminder/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.zoneminder" name="ZoneMinder" version="2.0.0" provider-name="Peter Gallagher">
+  <requires>
+    <import addon="xbmc.python" version="2.19.0"/>
```
Comment (enen92) [inline plugin.video.zoneminder/addon.xml]: 2.25.0

File: plugin.video.zoneminder/zoneminder.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+import sys
+from urllib import urlencode
+from urlparse import parse_qsl
+# Python3 equivalent
+# from urllib.parse import quote, parse_qsl
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+import requests
+import simplejson as json
```
Comment (enen92) [inline plugin.video.zoneminder/zoneminder.py]: import json

File: plugin.video.zoneminder/addon.xml (hunk 1 above)
Comment (petegallagher) [reply plugin.video.zoneminder/addon.xml]: removed

File: plugin.video.zoneminder/addon.xml (hunk 2 above)
Comment (petegallagher) [reply plugin.video.zoneminder/addon.xml]: updated

File: plugin.video.zoneminder/zoneminder.py (hunk 3 above)
Comment (petegallagher) [reply plugin.video.zoneminder/zoneminder.py]: updated

**Comment by petegallagher** [comment]
Thanks for the feedback, pull request updated accordingly.

**Pull request merged**